src/library_fs.js

```diff
@@ -1098,10 +1098,12 @@ FS.staticInit();` +
           if ((flags & {{{ cDefine('O_ACCMODE') }}}) !== {{{ cDefine('O_RDONLY') }}}) {
             trackingFlags |= FS.tracking.openFlags.WRITE;
           }
-          FS.trackingDelegate['onOpenFile'](path, trackingFlags);
+          var stat = FS.stat(path);
+          var fileSize = stat.size;
+          FS.trackingDelegate['onOpenFile'](path, trackingFlags, fileSize);
```

Comment:
You'll probably need to pass `flags` as well to generate a correct call to `open`.

Reply:
Will just replace `trackingFlags` with `flags`